mautic/mautic
# Focus: Add resize of modals and add scrolling If need it

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Fix the Focus modal and notification for mobile devices. 
This allows a scrollbar if the modal is larger than the device window.
Also, add the ability to change the size of the modal if the window is resized.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create focus  modal with image, text and form. Should be large
3. Add it to page
4. See If popup modal is displayed
5. Resize window to smaller size lik height of modal
6.  You should see scrollbar and you should able to submit form properly
